app/models/person.rb

```diff
@@ -222,25 +235,35 @@ def all_languages
   end
 
   def original_works
-    Manifestation.all_published.joins(expression: [work: :creations]).where("creations.person_id = #{self.id}")
+    Manifestation.all_published
+                 .joins(expression: [work: :involved_authorities])
+                 .where(involved_authorities: { person_id: id })
```

Comment:
wouldn't this include translations, too? This is supposed to return only Manifestations where this person is NOT the translator.

Reply:
Well... I believe this will not include translators, because translator can only be specified on Expression level (we have validation for this). But this will include all works where Person has any role possible on Work level (e.g. illustrator).

I wonder if we should limit it to return only works where person is an author? In this case I'd propose to remove this method at all and use new `Person.published_manifestations(:author)` method instead. @abartov  WDYT?